addons/l10n_id/views/res_bank.xml

```diff
@@ -6,7 +6,7 @@
         <field name="model">res.partner.bank</field>
         <field name="inherit_id" ref="base.view_partner_bank_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='currency_id']" position="after">
+            <xpath expr="//sheet/group/group[2]" position="inside">
```

Comment:
Technically it will work, but why not just add a new group in the //sheet/group for QRIS directly and avoid risking it being inheritred/...